tdrs-backend/tdpservice/users/migrations/0042_user_feature_flags.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2025-01-21 04:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0041_users_digit_group_add_datafile_permission'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='feature_flags',
+            field=models.JSONField(blank=True, default=dict, help_text='Feature flags for this data file.', null=True),
```

Comment:
Maybe a tooltip here to be explicit about formatting might help our admins:
```suggestion
            field=models.JSONField(blank=True, default=dict, help_text='Feature flags for this data file. JSON formatting example: {"fra_access": "True"}', null=True),
```